h/views/activity.py

```diff
@@ -174,6 +183,15 @@ def domain(user):
     if request.authenticated_user == user:
         result['user']['edit_url'] = request.route_url('account_profile')
 
+    if not result.get('q'):
+        if request.authenticated_user == user:
+            result['zero_message'] = _("You haven't made any annotations yet.")
```

Comment:
This is the one that will be getting replaced with something more elaborate, in a following PR